pkgs/tools/system/rofi-systemd/default.nix

```diff
@@ -3,13 +3,13 @@
 
 stdenv.mkDerivation rec {
   pname = "rofi-systemd";
-  version = "0.1.1";
+  version = "1.0.0";
```

Comment:
Where is the tag? https://github.com/colonelpanic8/rofi-systemd/tags

Reply:
oops, could swear I pushed that.